docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W --keep-going -n -T
```

Comment:
I'll revert this - but I had to exclude this flag (it's fail-on-warning) to be able to build the docs.

Reply:
What's the warning?

Reply:
"sphinx.errors.SphinxWarning: /Users/vladimir.kolovski/repos/mlflow-vlad/mlflow/mlflow/sklearn/__init__.py:docstring of mlflow.sklearn.autolog:130:duplicate label post training metrics, other instance in /Users/vladimir.kolovski/repos/mlflow-vlad/mlflow/docs/source/python_api/mlflow.pyspark.ml.rst
"

Reply:
Got it - thanks!